tasks/selinux.py

```diff
@@ -6,8 +6,8 @@
 import invoke
 from invoke import task
 
-DEFAULT_SYSPROBE_POLICY_TE_FILE = os.path.join(".", "cmd", "agent", "selinux", "system_probe_policy.te")
-DEFAULT_SYSPROBE_POLICY_OUTPUT_DIRECTORY = os.path.join(".", "cmd", "agent", "selinux")
+DEFAULT_SYSPROBE_POLICY_TE_FILE = os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "cmd", "agent", "selinux", "system_probe_policy.te")
+DEFAULT_SYSPROBE_POLICY_OUTPUT_DIRECTORY = os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "..", "cmd", "agent", "selinux")
```

Comment:
we ended up with two `".."`, that doesn't look right, right?

Reply:
Oops yeah I forgot to remove one of them